R/checks.R

```diff
@@ -789,4 +789,59 @@ check_fracwith_diswith <- function(form, rep) {
     filter(ptstatus == 1 & (num_of_fractures > 1 |
                               location_of_fracture != location_of_dislocation))
   return(problems)
+}
+
+#' Check if the number of fractures, if any, is valid for the set form5.x
+#' Also considers the logic of related selections for this check
```

Comment:
What are those related logic? You don't need to specify them in demo but we should know about them here.